documentation/src/docs/getting-started/pravega-on-kubernetes-101.md

```diff
@@ -17,6 +17,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+import versions from '@site/versions';
```

Comment:
@RaulGracia & @SrishT -- I've updated the "Deploy on K8s" docs to include the latest instructions as well as the latest versions. Versions are substituted in from here: https://github.com/pravega/docs/pull/1/commits/cf71c4b29058a16ff897c3a14cbccc4577d62e0d#diff-21a0578522c567f65881e4e8dc3aaace32d60dbdabf2730082b7d9da9f6cb4d0